Sources/DemoFeature/Demo.swift

```diff
@@ -116,6 +116,11 @@ public let demoReducer = Reducer<DemoState, DemoAction, DemoEnvironment>.combine
           userDefaults: $0.userDefaults
         )
       }
+    )
+    .pullback(
+      state: \DemoState.step,
+      action: /.self,
+      environment: { $0 }
```

Comment:
I have to do a pullback along the case path followed by the key path to get everything working, but once we have full optional paths we should be able to go back to the single pullback.